backend/installGlobalHook.js

```diff
@@ -31,29 +30,17 @@ function installGlobalHook(window: Object) {
           // but might add 2 (PROFILE) in the future.
           return 'development';
         }
-        // The above should cover envification, but we should still make sure
-        // that the bundle code has been uglified.
-        var findFiberCode = toString.call(renderer.findFiberByHostInstance);
-        // Filter out bad results (if that is even possible):
-        if (findFiberCode.indexOf('function') !== 0) {
-          // Hope for the best if we're not sure.
-          return 'production';
-        }
-
-        // By now we know that it's envified--but what if it's not minified?
-        // This can be bad too, as it means DEV code is still there.
-
-        // FIXME: this is fragile!
-        // We should replace this check with check on a specially passed
-        // function. This also doesn't detect lack of dead code elimination
-        // (although this is not a problem since flat bundles).
-        if (findFiberCode.indexOf('getClosestInstanceFromNode') !== -1) {
-          return 'unminified';
-        }
```

Comment:
This change means DevTools won't quite support 16 alpha/beta/RC? I guess that's okay though.

Reply:
It won't be as good at detecting unminified alphas. Which seems okay. It won't report false positives though.